src/main/java/com/drinchev/projectlabel/resources/ui/ProjectLabelBackgroundImage.java

```diff
@@ -109,11 +114,6 @@ private void setImageToIDE() {
         }
     }
 
-    private PropertiesComponent projectLevelPropertiesComponent() {
-        PropertiesComponent prop = project.getService(PropertiesComponent.class);
```

Comment:
Hi @simschla , as this might be the potential crash reason, described in #26 . I decided to get rid of this method and use the getter directly.

Reply:
LGTM, although the actual fix is probably not removing the method but checking `project.isDisposed` beforehand 😉

Reply:
I agree, however the return type is `PropertiesComponent` and in case the project is disposed I prefer to stop executing the flow further.

```
private PropertiesComponent projectLevelPropertiesComponent() {
      if (project.isDisposed()) {
            return; // Return what?
      }
      PropertiesComponent prop = project.getService(PropertiesComponent.class);
      return prop;
```

Reply:
Of course if you have a better suggestion let me know.